dnf-docker-test/features/steps/test_behave.py

```diff
@@ -274,13 +363,34 @@ def then_package_state(context, std_message, state, line_start):
                              .format(state))
 
 
-@then('file "{path_to_file}" should be "{state}"')
-def then_file_presence(context, path_to_file, state):
-    file_existence = os.path.isfile(path_to_file)
+@then('"{object}" "{path_to_object}" should be "{state}"')
```

Comment:
there seems to be much code for such easy function. Reduce the lines so it should support:

@then('"Path {path}" should be "{state}"')

state = present / absent
path = ".../bla/" (dir) / ".../bla" (file) / ".../bla/*" (non-empty `bla` dir)